src/api/submitOrder.js

```diff
@@ -55,7 +57,11 @@ module.exports = async (dvf, orderData) => {
     }
   }
 
-  return post(dvf.config.api + '/v1/trading/w/submitOrder', {
-    json: await dvf.createOrderPayload(value)
-  })
+  const json = await (
+    value.length
+      ? Promise.all(value.map(o => dvf.createOrderPayload(o)))
```

Comment:
can use `aigle` `map` or `mapSeries`

Reply:
Aigle is not yet added on dvf-client-js and wouldn't want to add it as new dependency unless necessary :)

Reply:
sorry was wrong it is there missed it :D

Reply:
btw. `aigle` has a really small footprint, it has 0 external deps